container/prometheus/alert_rules.yml

```diff
@@ -74,4 +74,31 @@ groups:
       severity: "critical"
     annotations:
       summary: "Snapmirror [{{ $labels.relationship_id }}] has [{{$value}}] lag time (in secs)"
-      description: "Snapmirror [{{ $labels.relationship_id }}] has [{{$value}}] lag time (in secs)"
\ No newline at end of file
+      description: "Snapmirror [{{ $labels.relationship_id }}] has [{{$value}}] lag time (in secs)"
+
+  # Object created. Refer https://netapp.github.io/harvest/latest/plugins/#changelog-plugin for more details.
+  - alert: Object Created
+    expr: change_log{op="create"} > 0
+    labels:
+      severity: "info"
+    annotations:
+      summary: "{{ $labels.object }} [{{ $labels.volume }}] created"
+      description: "{{ $labels.object }} [{{ $labels.volume }}] created"
```

Comment:
not all object would have volume label.
May be target Alert for volume only with additional filter of `object=volume`